schema.sql

```diff
@@ -0,0 +1,29 @@
+CREATE DATABASE doingsdone_db
+    DEFAULT CHARACTER SET utf8
+    DEFAULT COLLATE utf8_general_ci;
+
+USE doingsdone_db;
+
+CREATE TABLE users (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(128) NOT NULL UNIQUE,
+    email VARCHAR(128) NOT NULL UNIQUE,
+    password VARCHAR(255) NOT NULL,
+    date TIMESTAMP DEFAULT CURRENT_TIMESTAMP
```

Comment:
I recommend to create couple fields for every entity tables: created_at and updated_at. Db should automatically fill out created_at when you add row. you already did it. updated_at should be filled when system changes row. this case also possible.

help https://medium.com/@bengarvey/use-an-updated-at-column-in-your-mysql-table-and-make-it-update-automatically-6bf010873e6a

Reply:
Thanks, cool idea, I'll do it.